data.json

```
@@ -2199,6 +2199,17 @@
                 "JavaScript"
             ],
             "description": "a react-based framework that provides accessibility, modularity, responsiveness, and theming in a tidy package"
+        },
+        {
+            "name": "shivani",
```

```suggestion
            "name": "BookStore",
```

The name section refers to the name of the project being linked to.